data/hai/hai reveal 4 middle.txt

```diff
@@ -966,8 +966,30 @@ mission "Hai Reveal [A08] Symbol Identification"
 		log "Giti identified the angry cat symbol. It is from a long-defunct subversive group within the Syndicate called Base Principles. Somehow they obtained alien technology, and used the knowledge gained from it to advance their influence within the organization. A half-century ago, Syndicate Internal Affairs scoured them from the Syndicate's ranks in an extremely thorough and prejudiced inquisition."
 		
 		conversation
-			`After landing, you search up the way to the adjunct research library at the Navy Intelligence Observatory in Moscow. It takes three shuttle journeys and one high-speed rail trip, but you reach the building in under three hours. For someone of your background there are parts of this trip that seem dizzyingly unreal. As well traveled as you now are, actually navigating the urban environments of <origin> away from any particular spaceport is an experience not quite like any other. Once you reach the building in question you take a moment at the reception desk to interpret 'room number: 3017A19' - the 301st floor, section 7A, room 19 - and get directions. It's amazing how large Earth's megabuildings have become. Some planets have smaller populations than the building you are standing in right now, and it is but one of hundreds you've seen on your short journey.`
+			`You land at Ulaanbaatar's spaceport and receive a message that Batu has to finalize some arrangements and will arrive in the evening. That leaves plenty of time for you to see to your business of angry cat symbols.`
+			`	At an information desk, you inquire about the route to the library, and are given two options. You can take a very expensive private hovertaxi service, which will be payed for by the Syndicate. Or, you can take the long route and see the sights. Either way, you'll have plenty of time before Batu arrives.`
+			choice
+				`	(Take the direct route.)`
+					goto "take the direct route"
+				`	(Enjoy the sights along the way.)`
+			`	Moscow is thousands of kilometers away, so you travel most of the route on Shuttles. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but not as exciting as your experiences.`
```

Comment:
```suggestion
			`	Moscow is thousands of kilometers away, so you travel most of the route on shuttlecraft. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but they'll never experience space like you have.`
```

Reply:
The change to the last sentence doesn't work. Those pilots may experience space like you have, but not until they can afford a new ship and leave their exclusive contract.

Reply:
```suggestion
			`	Moscow is thousands of kilometers away, so you travel most of the route on Shuttles. It seems some new pilots took an option you never considered: planet-side travel in an exclusive contract with local transit authorities who pay half the mortgage. Certainly a safe career, but it's too dull to be restrained to just one planet.`
```
I see what you mean. Is this better?

Reply:
I have applied those changes. I haven't seen any others though.

Reply:
Yeah, you applied the right ones, thanks.